DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Package/WebApiLibraryPackage.cs

```diff
@@ -10,6 +14,10 @@ public void RegisterServices(Container container)
         {
             container.Register<IGithubService, GithubService>(Lifestyle.Singleton);
             container.Register<IAuthService, AuthService>(Lifestyle.Scoped);
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies().ToList();
+            container.Register(typeof (IValidator<>), assemblies, Lifestyle.Singleton);
+            container.RegisterConditional(typeof (IValidator<>), typeof (NullValidator<>), Lifestyle.Singleton,
```

Comment:
I restored previous way of registration, because `new[] { typeof(IValidator<>).Assembly };` return only FluentValidation library, that doesn't include validators, that we have created.